docs/recipes/MigratingToMMKV.md

```diff
@@ -15,22 +15,25 @@ publish_date: 2022-10-28
 
 [MMKV](https://github.com/mrousavy/react-native-mmkv) is said to be the fastest key/value storage for React Native. It has encryption support for secure local storage and also uses synchronous storage to simplify your application code.
 
-In this recipe, we'll convert our the Ignite demo project from using `AsyncStorage` to `MMKV`:
+In this recipe, we'll convert our the Ignite demo project from using `AsyncStorage` to `MMKV`.
+
+We'll get started by igniting a new application with the `prebuild` workflow. We must do this since `react-native-mmkv` contains native dependencies not included in the Expo SDK. Luckily with Ignite CLI, it's easy to jump into this workflow:
```

Comment:
nit: I think we decided on using `cng` verbiage instead of `prebuild` - so we might want to be consistent here

Reply:
Good call!